src/main/java/com/icthh/xm/ms/entity/service/FunctionExecutorService.java

```diff
@@ -1,58 +0,0 @@
-package com.icthh.xm.ms.entity.service;
-
-import com.icthh.xm.commons.lep.LogicExtensionPoint;
-import com.icthh.xm.commons.lep.spring.LepService;
-import com.icthh.xm.commons.logging.aop.IgnoreLogginAspect;
-import com.icthh.xm.ms.entity.domain.ext.IdOrKey;
-import com.icthh.xm.ms.entity.lep.keyresolver.FunctionLepKeyResolver;
-import java.util.Map;
-
-/**
- * The {@link FunctionExecutorService} interface.
- */
-@IgnoreLogginAspect
-@LepService(group = "function")
-public class FunctionExecutorService {
```

Comment:
can we added smth line proxy for this class to keep spring bean name and simplify migration

Reply:
done